### PR TITLE
Tweaks

### LINKED ISSUE #167: Controversial Content section should always be expanded if new reader mode is off and some options are checked

### PR DESCRIPTION
**Pull request by GiovanH**
Resolves #167, #168, #172, #174, #179

Also uses URL as tab title fallback instead of the home page title

### CODE REVIEW

**Comment by GiovanH** [comment]
image

**Comment by GiovanH** [comment]
7ba3fd476b0be7011470925f7f0ce43a25ce4243 consolidates the footnote logic like I probably should have done to begin with

**Comment by GiovanH** [comment]
image

File: src/components/AppMenu/TabBar.vue [hunk 1]
```diff
@@ -3,11 +3,48 @@
...
+              v-if="$localData.settings.devMode && thisTabPageId" 
+              :title="`Change current page from ${$newReaderCurrent} to p=${thisTabPageId}\n(Middle click to disable newreader)`"
+              :class="{active: thisTabPageId != $newReaderCurrent}"
+              @click="$updateNewReader(thisTabPageId, true)" 
+              @click.middle="$localData.root.NEW_READER_CLEAR">
+              <fa-icon icon="lock"></fa-icon>
+              <span class="badge" v-text="$newReaderCurrent"></span>
+            </div>
+            <div class="systemButton"
+              @click="isCurrentPageBookmarked ? tabComponent.$refs.bookmarks.open() : tabComponent.$refs.bookmarks.newSave()"
```
Comment (Bambosh) [inline src/components/AppMenu/TabBar.vue:43]: Maybe make this one a click-to-toggle? Currently does nothing if you try to click while the menu is open

File: src/components/UIElements/Bookmarks.vue [hunk 2]
```diff
@@ -201,293 +206,303 @@ export default {
...
+
+    .bookmarkShadow {
+      position: fixed;
+      top: 0;
+      left: 330px;
+      width: 100%;
+      height: 100%;
+
+      pointer-events: all;
+    }
```
Comment (Bambosh) [inline src/components/UIElements/Bookmarks.vue:292]: If we're going in this direction we might wanna darken the entire page to indicate it's unclickable. That said, I'm not sure how useful this is as compared to letting people navigate the page while this is open. I figure the kinda person that has a long list of bookmarks might appreciate it if they're trying to cross-reference stuff between pages.

File: src/components/AppMenu/TabBar.vue (hunk 1 above)
Comment (GiovanH) [reply src/components/AppMenu/TabBar.vue:43]: It bookmarks the current page if it isn't already. It also opens the bookmarks pane if it isn't open already.

**Comment by GiovanH** [comment]
dee8089 isn't working; can't edit bookmarks without the pane collapsing

**Review by Bambosh — APPROVED** [review]

**Pull request merged**